Bottom: eda812c3fe905cc5bf944a532076ed9dac8aad3b
Top:    cedfe9ac3ea5ee0d65c541170388d207f25fdeb2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-04 17:16:11 +0000

Just make a black background.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 6e3e9f0..04be0ff 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1948,9 +1948,9 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
 #endif
 
   glEnable (GL_STENCIL_TEST);
-  glClearColor (port->offlimits_color.red / 65535.,
-                port->offlimits_color.green / 65535.,
-                port->offlimits_color.blue / 65535.,
+  glClearColor (port->bg_color.red / 65535.,
+                port->bg_color.green / 65535.,
+                port->bg_color.blue / 65535.,
                 1.);
   glStencilMask (~0);
   glClearStencil (0);
@@ -2053,6 +2053,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   /* Drawing operations as masked to areas where the stencil buffer is '0' */
   /* glStencilFunc (GL_GREATER, 1, 1); */           /* Draw only where stencil buffer is 0 */
 
+#if 0
   if (global_view_2d) {
     glBegin (GL_QUADS);
     glVertex3i (0,             0,              0);
@@ -2083,6 +2084,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
     }
     glEnd ();
   }
+#endif
 
   ghid_draw_bg_image ();
